src/ServicePulse.Host/app/modules/monitoring/js/endpoint_details.controller.js

```diff
@@ -72,7 +74,9 @@
                         }
                     }, function (err) {
                         // Warn user?
-                    });
+                        });
+
+                    $scope.endpoint.isStale = $scope.endpoint.isStale && instance.isStale;
```

Comment:
Should this not be an `||`? Is an endpoint stale only if all the instances are stale or is it stale if at least one instance is stale?

Reply:
The endpoint is stale if all instances are stale => `isStale` is true for all instances => `&&`.

Reply:
:+1: